mxcube3/routes/SampleCentring.py

```diff
@@ -24,12 +24,22 @@ def init_signals():
             mxcube.diffractometer.connect(mxcube.diffractometer, signal, signals.task_event_callback)
         else:
             pass
+    try:
+        frontlight_hwobj = mxcube.diffractometer.getObjectByRole('frontlight')
+        frontlight_hwobj.connect(frontlight_hwobj, 'positionChanged', signals.motor_event_callback)
+        frontlightswitch_hwobj = mxcube.diffractometer.getObjectByRole('frontlightswitch')
+        frontlightswitch_hwobj.connect(frontlightswitch_hwobj, 'actuatorStateChanged', signals.motor_event_callback)  
+    except Exception:
+        logging.getLogger('HWR').exception('[SAMPLEVIEW] front light hwobj error')
+
+    try:
+        backlight_hwobj = mxcube.diffractometer.getObjectByRole('backlight')
+        backlight_hwobj.connect(backlight_hwobj, 'positionChanged', signals.motor_event_callback)
+        backlightswitch_hwobj = mxcube.diffractometer.getObjectByRole('backlightswitch')
+        backlightswitch_hwobj.connect(backlightswitch_hwobj, 'actuatorStateChanged', signals.motor_event_callback)
+    except Exception:
+        logging.getLogger('HWR').exception('[SAMPLEVIEW] back light hwobj error')
```

Comment:
We need to do the same kind of trickery as in `Login.py`, `get_initial_state()` : we (at ESRF) do not have 2 separate objects for backlight and frontlight (I wonder why you separated them, by the way... ?). I can have a look and propose a change, if you want ? It's going to be something like what you proposed yesterday for the front/back light in `Login.py`.